msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -19,14 +31,12 @@
     <Summary>This package contains the binaries of the Microsoft Authentication Library (MSAL). MSAL makes it easy to obtain tokens from Azure AD v2 (work &amp; school accounts, MSA) and Azure AD B2C, gaining access to Microsoft Cloud API and any other API secured by Microsoft identities. This version supports adding authentication functionality to your .NET based client on Windows desktop (.NET 4.5+), UWP, .NET Core, Xamarin iOS and Xamarin Android.</Summary>
     <Copyright>(C) Microsoft Corporation. All rights reserved.</Copyright>
     <PackageTags>Microsoft Authentication Library;MSA;MSAL;B2C;Azure Active Directory;AAD;Identity;Authentication;.NET;Windows Store;Xamarin iOS;Android</PackageTags>
-    <Product>Microsoft.Identity.Client ($(TargetFramework))</Product>
+    <Product>Microsoft.Identity.Client</Product>
     <IncludeSymbols>true</IncludeSymbols>
-    <GenerateDocumentationFile Condition=" '$(Configuration)' == 'Release' ">true</GenerateDocumentationFile>
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>..\..\build\MSAL.snk</AssemblyOriginatorKeyFile>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <!-- Workaround until Xamarin supports PPDB -->
     <UseFullSemVerForNuGet>false</UseFullSemVerForNuGet>
-    <DebugType>full</DebugType>
+    <PackageVersion>1.1.5</PackageVersion>
```

Comment:
Versioning is done by changing this property

Reply:
Awesome! Will add this to our release notes.